File: BUEN_SABOR_FRONTEND/src/models/ArticuloManufacturado.ts
import Articulo from "./Articulo";
import DetalleArticuloManufacturado from "./DetalleArticuloManufacturado";
export default class ArticuloManufacturado extends Articulo {
    descripcion: string = "";
    ganancia: number = 0;
    tiempoEstimadoMinutos: number = 0;
    preparacion: string = "";
    detalles: DetalleArticuloManufacturado[] = [];
}

File: BUEN_SABOR_BACKEND/src/main/java/com/lab4/buen_sabor_backend/model/ArticuloManufacturado.java
package com.lab4.buen_sabor_backend.model;

import com.fasterxml.jackson.annotation.*;
import jakarta.persistence.*;
import lombok.*;

import java.util.*;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@DiscriminatorValue("ArticuloManufacturado")
public class ArticuloManufacturado extends Articulo {

    private String descripcion;
    private Integer tiempoEstimadoMinutos;
    private String preparacion;

    @OneToMany(mappedBy = "articuloManufacturado", cascade = CascadeType.ALL, orphanRemoval = true)
    private Set<DetalleArticuloManufacturado> detalles = new HashSet<>();

}

